components/mdns/mdns_networking_socket.c

```diff
@@ -27,6 +27,13 @@
 #include <net/if.h>
 #endif
 
+#if defined(CONFIG_IDF_TARGET_LINUX)
+#define SOCKET_IPV4 1
+#define SOCKET_IPV6 CONFIG_LWIP_IPV6
+#else
+#define SOCKET_IPV4 LWIP_IPV4
+#define SOCKET_IPV6 LWIP_IPV6
+#endif
```

Comment:
Again, need to find a better way of IPv4/6 config

Reply:
Removed